scripts/overloads_no_missing.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+
+import libcst as cst
+import libcst.matchers as m
+from libcst import codemod
+
+EllipsisType = type(Ellipsis)
```

Comment:
Is using `EllipsisType` from the types module not possible here?

Reply:
we support python3.8 development environments, even if we don't need to, it makes the development process easier.